app/backend/templates/system/reference_report.md

```diff
@@ -26,6 +26,7 @@ Name: {{ reference.from_user.name }}
 Email: {{ reference.from_user.email }}
 Username: {{ reference.from_user.username }}
 User ID: {{ reference.from_user.id }}
+Private Text: {{ reference.from_user.private_text }}
```

Comment:
@aapeliv I am not 100% this is right so please correct me if not! I added it to one of the backend tests though and it's passing so I hope so?